src/loaders/ImageLoader.js

```diff
@@ -77,12 +77,20 @@ ImageLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 		}
 
+		if ( this.priorityHint ) image.importance = this.priorityHint;
+
 		scope.manager.itemStart( url );
 
 		image.src = url;
 
 		return image;
 
+	},
+
+	setPriorityHint: function ( priorityHint ) {
```

Comment:
How about `setImportance`?

Reply:
...that, or `setImportanceAttribute()` -- which I think is clearer, given that what it is it is not very clear to begin with.

Reply:
I'm ok with that. The only hesitation I have is that "importance" doesn't really clue people in to what it does. You can search for "priority hint" and end up at a w3c working group, but importance may be harder to search for.

Reply:
FWIW, from https://wicg.github.io/priority-hints/

 > The Priority Hints feature ... introduces the importance attribute.

    <img src="img/carousel-1.jpg" importance="high">

>but importance may be harder to search for.

The documentation should explain this method, so searching should not be required.

Either name is fine with me.